src/main/java/bossmonster/exception/Validator.java

```diff
@@ -0,0 +1,67 @@
+package bossmonster.exception;
+
+import bossmonster.domain.Player;
+
+import java.awt.print.PrinterIOException;
+import java.util.StringTokenizer;
+
+public class Validator {
+
+    final String PREFIX = "[ERROR] ";
+
+    public void validateBossStatus(String hp) {
+        if (!hp.chars().allMatch(Character::isDigit)) {
+            throw new IllegalArgumentException(PREFIX + "보스 체력은 100이상, 300이하의 정수로 입력해주세요.");
+        }
+
+        int hpNum = Integer.parseInt(hp);
+        if (hpNum < 100 || hpNum > 300) {
+            throw new IllegalArgumentException(PREFIX + "보스 체력은 100이상, 300이하여야합니다.");
+        }
+    }
+
+    public void validatePlayerName(String playerName) {
+        if (playerName.length() > 5) {
+            throw new IllegalArgumentException(PREFIX + "플레이어의 이름은 5자 이하만 가능합니다.");
+        }
+    }
+
+    public void validatePlayerStatus(String status) {
+        StringTokenizer st = new StringTokenizer(status, ",");
+        String hp = st.nextToken();
+
+        if (!st.hasMoreTokens()) {
+            throw new IllegalArgumentException(PREFIX + "HP와 MP를 콤마(,)로 구분하여 입력해주세요. (예: 100,100)");
+        }
+
+        String mp = st.nextToken();
+
+        if (!hp.chars().allMatch(Character::isDigit) || !mp.chars().allMatch(Character::isDigit)) {
```

Comment:
조금 더 간단히 처리할 순 없을까요?

Reply:
입력받은 문자열이 숫자가 아닌 문자나, 띄어쓰기가 있을 때 어떻게 해결할지를 찾아봤을 때 해당 방법이 있어서 사용했었는데, 조금 더 간단히 처리한다면 정규식을 이용한 방법일까요?

Reply:
정규식도 있을 수 있겠고, 정말 간단히는 다음과 같이도 가능할 것 같아요!
```suggestion
validateDigit(hp);
validateDigit(mp);

// ===============

private void validateDigit(String hp) {
    try {
         return Integer.parseInt(hp);
    } catch(NumberFormetException e) {
        // 예외처리 메세지
    }
}
```

Reply:
아.. NumberFormetException이 뜨길래 어떻게 처리할까 고민했었는데 저렇게 하는 방법도 있었겠네요..! 참고할게요 감사합니다!